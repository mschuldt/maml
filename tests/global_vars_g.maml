arduino = Arduino(desktop=True)

@block(once)
def test():
    a = 1
    b = 2
    c = 3
    print(a)
    print(b)
    print(c)
    print(a+b+c)
    die(0)

arduino.send(test)
