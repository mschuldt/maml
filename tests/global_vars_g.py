arduino = Arduino(desktop=True)

@block
def test():
    a = 1
    b = 2
    c = 3
    print_i(a)
    print_i(b)
    print_i(c)
    print_i(a+b+c)
    die(0)

arduino.send(test)
