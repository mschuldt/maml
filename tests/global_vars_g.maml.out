Initializing avm...
Ready.

1
2
3
6
bye.
